cmd/nerdctl/volume_ls.go

```diff
@@ -54,220 +45,63 @@ func newVolumeLsCommand() *cobra.Command {
 	return volumeLsCommand
 }
 
-type volumePrintable struct {
-	Driver     string
-	Labels     string
-	Mountpoint string
-	Name       string
-	Scope      string
-	Size       string
-	// TODO: "Links"
-}
-
 func volumeLsAction(cmd *cobra.Command, args []string) error {
+	options := &types.VolumeLsCommandOptions{}
+	options.Writer = cmd.OutOrStdout()
 	quiet, err := cmd.Flags().GetBool("quiet")
 	if err != nil {
 		return err
 	}
-	volumeSize, err := cmd.Flags().GetBool("size")
+	options.Quiet = quiet
```

Comment:
nit: instead of storing in local variables everytime, you can do

```golang
options.Quiet, err := cmd.Flags().GetBool("quiet")
...
```

same for other options.

Reply:
another style I recommand is first storing all flags in local variables and at the end, create the option struct

```golang
format, err := cmd.Flags().GetBool("format")
...
quiet,  err := cmd.Flags().GetBool("quiet")
...

option := types.VolumeLsCommandOptions{
    Format: format,
    Quiet: quiet,
}
```